scripts/GetClrDbg.ps1

```diff
@@ -104,13 +104,14 @@ if (-not $RuntimeID) {
 }
 Write-Host "Info: Using Runtime ID '$RuntimeID'"
 
-# create the install folder if it does not exist
-if (-not (Test-Path -Path $InstallPath -PathType Container)) {
-    New-Item -ItemType Directory -Force -Path $InstallPath
+# create the temp folder if it does not exist
+$TempPath = Join-Path -Path $env:TEMP -ChildPath "getclrdbg"
+if (-not (Test-Path -Path $TempPath -PathType Container)) {
+    New-Item -ItemType Directory -Force -Path $TempPath
 }
```

Comment:
I don't believe so. 

---
In reply to: [63795897](https://github.com/Microsoft/MIEngine/pull/332#discussion_r63795897) [](ancestors = 63795897)